src/services/auth.service.js

```diff
@@ -0,0 +1,35 @@
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
```

Comment:
The `secure` option in the nodemailer transport configuration is set to `true`. Ensure that your SMTP server supports and requires a secure connection. If not, you might need to set this to `false`.